model/file_info.go

```diff
@@ -6,72 +6,169 @@ package model
 import (
 	"bytes"
 	"encoding/json"
+	"image"
 	"image/gif"
 	"io"
 	"mime"
 	"path/filepath"
+	"strings"
 )
 
 type FileInfo struct {
-	Filename        string `json:"filename"`
-	Size            int    `json:"size"`
+	Id              string `json:"id"`
+	UserId          string `json:"user_id"`
```

Comment:
We either use UserId or CreatorId for the owner depending on which file we're in. I think I was working off of Post when I filled in these fields so it ended up UserId. I'd prefer to use CreatorId instead of CreatedByUserId since it's shorter and consistent with some of the others, but I'm 0/5 on that vs UserId